scripts/sbuild-it

```diff
@@ -15,6 +15,7 @@ Usage: ${0##*/} [ options ] dsc [arch [ release ] ]
     -h | --help             display this message
          --dry-run          only report what would be done
          --no-arch-all      do not pass '--arch-all'
+         --pt OPT           pass the argument through to sbuild.
```

Comment:
Not clear to me what `pt` stands for? Also worth adding that `--pt` can be specified multiple times.

Reply:
--pt was 'pass-through' (the commit message did say that).
I changed it to  --pass-through / -p.